ops-bedrock/entrypoint-l1.sh

```diff
@@ -14,9 +14,9 @@ WS_PORT="${WS_PORT:-8546}"
 
 if [ ! -d "$GETH_KEYSTORE_DIR" ]; then
 	echo "$GETH_KEYSTORE_DIR missing, running account import"
-	echo -n "pwd" > "$GETH_DATA_DIR"/password
-	echo -n "$BLOCK_SIGNER_PRIVATE_KEY" | sed 's/0x//' > "$GETH_DATA_DIR"/block-signer-key
-	geth account import \
+	printf "%s" "pwd" > "$GETH_DATA_DIR"/password
```

Comment:
I'm unclear why this is a better version.  `echo -n` seems simpler and easier to understand with less chance of error in getting the format string right.

Reply:
This one we can move to `/bin/bash` safely and then use `echo -n` (the op-geth docker image does indeed have bash).

Reply:
Perfect, updated to bash in ba7d2092e

Reply:
ba7d2092e resulted in CI failing, as the `ethereum/client-go:v1.13.5` image used does not contain bash
```
 ⠿ Container ops-bedrock-l1-1    Starting 0.7s
Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown
```

@ajsutton Is there a similar image available that does have bash we should switch to, or should we revert ba7d209 and stick with `printf` for now?

Reply:
There isn't one such image. Though we should install bash ourselves when [building the image](https://github.com/ethereum-optimism/optimism/blob/develop/ops-bedrock/Dockerfile.l1)

Reply:
Pushed 01d5849 which installs bash, and `make devnet-up` now hangs. I killed the CI job after an hour and restarted it to see if it was a flake, but both times it ends up stuck at `[INFO|<timestamp>] Waiting for RPC server at 127.0.0.1:8545`. Locally, without docker, `make devnet-up` works as expected, so this feels related to that change

Reply:
Fixed in a6462d4, so resolving this thread